mycore-ocfl/src/main/java/org/mycore/ocfl/commands/MCROCFLCommands.java

```diff
@@ -57,6 +56,14 @@ public class MCROCFLCommands {
 
     public static final String FAILED_AND_NOW_INVALID_STATE = FAILED + " and now invalid state";
 
+    private static final MCROCFLXMLUserManager OCFL_USER_MANAGER
+        = MCRConfiguration2.<MCROCFLXMLUserManager>getSingleInstanceOf("MCR.OCFL.User.Manager")
+            .orElseThrow();
+
+    private static final MCROCFLXMLClassificationManager OCFL_CLASSIFICATION_MANAGER
+        = MCRConfiguration2.<MCROCFLXMLClassificationManager>getSingleInstanceOf("MCR.OCFL.Classification.Manager")
+            .orElseThrow();
+
```

Comment:
See comment by @yagee-de, fully agree here as well, it can get called when it's needed but I don't see the gain of using the before mentioned configs for the managers. Commands are not meant to be constantly ran with every operation, plus see comment on the MCROCFLXMLUserManager regarding performance impact of initializing it with `new MCROCFL[...]Manager()`